example/package.json

```diff
@@ -34,6 +34,7 @@
     "@babel/preset-react": "^7.0.0",
     "@testing-library/react": "^8.0.4",
     "babel-jest": "^24.8.0",
+    "bs-platform": "^6.0.3",
     "graphql-query-test-mock": "^0.11.1",
```

Comment:
it's inconvenient to have to install a different `bs-platform` globally especially since most likely people would have installed globally version 5